### ISSUE TITLE
sickchill don't start after last update

### ISSUE THREAD

**Issue by kretzu77** [BenjV/SYNO-packages#10]
Hi,

I have sickchill instaled on synology.
After last update the package don't start anymore.

Do you have a fix fot this please?

Best regards

**Comment by BenjV** [comment]
The SickChill developers added (by mistake I think) a few library folders with new libs that when used prevent SickChill to start.
The solution is to remove those libs via the command prompt as follow:

Log in via ssh to your Nas as admin.
Make yourself root with the following command
sudo -i 
Then remove the libs with the following commands.
rm /var/packages/sickchill/target/var/SickBeard/lib/feedparser
rm /var/packages/sickchill/target/var/SickBeard/lib/exceptions
rm /var/packages/sickchill/target/var/SickBeard/lib/lxml

After that SickChill will start again
Removing those folders could also be done with WinSCP

If you have trouble with this and in not to much of a hurry you can also wait .
I will try this week to make a new package which can update SickChill and remove those folders automatically.

**Comment by kretzu77** [comment]
Worked.
Thank you very much for your quick respond.

Respect

**Comment by BenjV** [comment]
I have created an new package version 1.2 which can be manually installed on your Nasses.
It will upgrade the existing SickChill package with SickChill code from their github (now without the wrong libs).
The config and database will be kept intct

**Comment by dericco** [comment]
Hi BenjV, thank you very much for the package. I was able to install and run it without issues, but it seems to be empty, i.e. my existing config and DB are not considered. 
Before I used the SickBeard Custom package by Diaoul, pointing to the SickChill git repository. Maybe its because I started with SickRage some years ago and all files are in different folders?
My previous installation had problems with logs not updating anyways, so maybe this is a good time to start fresh. If only I could remember all the settings I made 🙈

**Comment by BenjV** [comment]
So you did a fresh install with my SIckChill package and of course you get an empty SickChill.
Only if you already had a my SickChill package then the new version would automatically have done an upgrade and not a fresh install.

If you did not remove the sickbeard-custom package then you still have the database and config and you could copy them to the SickChill installation.
So stop SickChill via the package center.
Copy the sickbeard.db, cache.db and config.ini from SickBeard-Custom to SickChill
and start SickChil again.

Those files can be found here:

> /var/packages/sickbeard-custom/target/var

and should be copied to here:

> /var/packages/sickchill/target/var

**Comment by dericco** [comment]
Hey BenjV, that makes perfect sense and are exactly the info would have asked for next. Thank you! Will try that in the next days.

**Comment by musiczetetic** [comment]
I had this problem and when I tried to use the 1.2 update package I ran into errors, this coincided with other issues that prompted me to do a long overdue factory reset of my NAS. I have a fresh install of DSM 6.2.2 and I installed the 1.2 package but I get an error "Failed to run the package service"
Should I be using that package for a fresh install or is there another recommended way?

**Comment by BenjV** [comment]
I don't understand what you are saying?
If you did a factory factory reset then you had to do a fresh install of DSM and all the packages.

If it was a simple reset (just one beep) then save the files as I said in the previous post.
Change the port in the config.ini to 8899 and restore those files after the new installation.

If you want to save some extra time afterwards you could also save the subfolder "cache" with all its subfolders and files and restore them afterwards.

**Comment by musiczetetic** [comment]
Yes, I did a full OS reinstall. I have already written off my config and happy to start from scratch. I can't start Sickchill after a fresh install, I just get "Failed to run the package service" error.

**Comment by BenjV** [comment]
You could check a few tings.
 
Did you install python 2.7 and git from the SynoCommunity first?
Did you red this post? Just the first item the rest is rubibsh.

> https://github.com/SickChill/SickChill/issues/5821

This has to be done after you installed git from the SynoCommunity and before you install SickChill.
If I can find some time this weekend I will add it to the SickChill package and create a new .spk

If things still don't work you can look at the package install log, that can be found here:

> /var/packages/sickchill/target/var/sickchill_install.log

And in the log of the package center, that can be found here:

> /var/log/synopkg.log

And by the way, don't forget to give the group sc-download give read/write access to shared folder were you downloader (torrent or nzb) stores the downloaden files.

**Comment by musiczetetic** [comment]
It was the GIT packagd issue, thanks heaps.

**Comment by BenjV** [comment]
I have create a new SickChill package version 1.3 which correct the git problem with the certificate.
It can be found here:
https://github.com/BenjV/SYNO-packages/raw/master/SickChill%20noarch%20V1.3.spk

**Comment by Numiah** [comment]
Hmm. odd. I installed 1.3 after the updates kinda mauled my installation.
All is up and running, however I can't open any of the downloaded media files.
The ownership stays on sc-nzbget?
Wat was changed to get this as a result?

**Comment by BenjV** [comment]
There was no change at all.
If you do not move or copy the media files in the postprocessing of SickChill the media files will keep the ownership of the downloader in this case nzbget, ergo the ownership is sc-nzbget.

If you are accessing the media files with a user that has not enough privileges to open the file then you have to add that user to the group sc-download.
Again, this was not changed and was always the case.

**Comment by nadimalam** [comment]
> I have create a new SickChill package version 1.3 which correct the git problem with the certificate.
> It can be found here:
> https://github.com/BenjV/SYNO-packages/raw/master/SickChill%20noarch%20V1.3.spk

Hey, 

Please can you help, i installed this package but unfortunetly after trying various things throughout the whole day today i lost my most recent settings of Sickbeard that i had on my Synology NAS, as i tried uninstalling the previous package and reinstalling it hoping that would fix the issues that have been going on for last 2-3 months now. However, that removed all my previous files.

Luckily i had a previous backup of my settings from 2018, so i reimported the config.ini, cache.db and Sickbeard.db files as mentioned in one of your previous posts here.

Now thankfully most of my settings are back, however, ive lost all my tv shows that i had added and for some reason i just cant find myself being able to scan the folder which has all the previous tv shows and reimporting them into sickchill.

Please can you tell me how to do this? Ive tried Adding Shows from Folder, and it only finds like a few handful of shows, but even those when i click add it goes to the next screen which says to select a show and nothing is available to select, so i cant add anything!!

**Comment by BenjV** [comment]
Has the group sc-download read/write privileges on those folders.
To be sure use File Station to set those privileges for the top folder and all the subfolders and files.

**Comment by Numiah** [comment]
> Hmm. odd. I installed 1.3 after the updates kinda mauled my installation.
> All is up and running, however I can't open any of the downloaded media files.
> The ownership stays on sc-nzbget?
> Wat was changed to get this as a result?

Solved it. For some reason nzbget was keeping permissions.
Setting "${AppDir}/unrar x -ai" fixed it.
Not sure why the standards settings always worked until I updated SickChill.
Problems only occurred to tv shows, not for movies which are controlled from Sonarr.

**Comment by nadimalam** [comment]
> > I have create a new SickChill package version 1.3 which correct the git problem with the certificate.
> > It can be found here:
> > https://github.com/BenjV/SYNO-packages/raw/master/SickChill%20noarch%20V1.3.spk
> 
> Hey,
> 
> Please can you help, i installed this package but unfortunetly after trying various things throughout the whole day today i lost my most recent settings of Sickbeard that i had on my Synology NAS, as i tried uninstalling the previous package and reinstalling it hoping that would fix the issues that have been going on for last 2-3 months now. However, that removed all my previous files.
> 
> Luckily i had a previous backup of my settings from 2018, so i reimported the config.ini, cache.db and Sickbeard.db files as mentioned in one of your previous posts here.
> 
> Now thankfully most of my settings are back, however, ive lost all my tv shows that i had added and for some reason i just cant find myself being able to scan the folder which has all the previous tv shows and reimporting them into sickchill.
> 
> Please can you tell me how to do this? Ive tried Adding Shows from Folder, and it only finds like a few handful of shows, but even those when i click add it goes to the next screen which says to select a show and nothing is available to select, so i cant add anything!!

I think using an old .db and config file from over a year ago was not compatible with the new version of sickchill. I checked in the folder and there was dozens of different version of .db files created and numbered. I guess it messed up something, hence i just started a new installtion of Benjv sickchill, then spent the rest of the day just changing settings and setting up the tv shows again (cant believe a whole day wasted on this ever since the project broke), anyway just glad to be up and ready again, since sickbeard/sickchill havent been working for like 3 months.  Just hope this doesnt happen ever again!
